config/crd/bases/ipam.metal3.io_ipaddresses.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.11.4
+    controller-gen.kubebuilder.io/version: v0.12.1
```

Comment:
CAPI is sitting currently on v0.12.0. Its fine for now since its a patch version but lets wait for CAPI to bump this for future bumps.